Doc/library/math.rst

```diff
@@ -136,6 +136,15 @@ Number-theoretic and representation functions
    .. versionadded:: 3.5
 
 
+.. function:: lcm(a, b)
+
+   Return the least common multiple of integers *a* and *b*.  The value of
+   ``lcm(a, b)`` is the smallest nonnegative integer that is a multiple of
```

Comment:
@mdickinson
Is not 0 a multiple of any number?

Maybe clarify that it is the smallest *positive* integer for non-zero a and b?

Reply:
@serhiy-storchaka Yes, `0` is a multiple of any number, so to a mathematician, the first sentence is enough by itself. But in my experience, people get worried and uncertain when zeros are involved, so it's worth having the second sentence to clarify the behaviour in that case.

I'd prefer to keep the "nonnegative" language.

Reply:
I'm open to simply dropping the second sentence if everyone thinks it's clear without it. Maybe I'm worrying unnecessarily.

Reply:
Hmm. I don't remember the "nonzero" bit from before. That _is_ confusing, and should be removed.

Reply:
If we're fine-tuning the documentation, I'd also suggest adding an extra blank line between the first sentence and the second, to make it a bit clearer that that seconde sentence is simply informative and not part of the definition. Alternatively, the second sentence could be parenthesized. (Or again, simply removed.)